src/components/AppHeader.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Logo from './Logo';
+import Title from './Title';
+import styled from '@emotion/styled';
+import Manawallpaper from '../assets/manawallpaper.jpg';
+
+const Header = styled.header`
+  width: 100vw;
```

Comment:
`AppContainer`, the parent of this already has a `height` of `100vw`

Reply:
changed to width: 100%